packages/base/README.md

```diff
@@ -93,6 +93,14 @@ You might find yourself in a position where you want to handle *all* commands, i
 1. `cmdName` - string representing the command being run
 1. `...args`
 
+There is a bit of a gotcha with handling Appium commands. Appium drivers have the ability to turn on a special 'proxy' mode, wherein the Appium server process takes a look at incoming URLs, and decides whether to forward them on to some upstream WebDriver server. It could happen that a command which a plugin wants to handle is designated as a command which is being proxied to an upstream server. In this case, we run into a problem, because the plugin never gets a chance to handle that command! For this reason, plugins can implement a special member function called `shouldAvoidProxy`, which takes the following parameters:
+
+1. `method`
```

Comment:
I assume we need also to explain possible values for these params

Reply:
yeah good point